### DISCUSSION TITLE
Change which email to use for email notifications

### DISCUSSION THREAD

**Discussion by NicoSchwandner** [dailydotdev/daily#1535] [Feature Requests]
I have added several emails:

- Personal email: To associate my account forever with me, even when I change work
- Work email: To ideally receive daily digests and read through work-related articles

Unfortunately I always receive the daily digest emails on my personal email, but would like to receive them on my work email.

**Comment by idoshamun** [comment]
Hi @NicoSchwandner,

Thanks a lot for the suggestion. We will consider it :)
